numpy/distutils/mingw32ccompiler.py

```diff
@@ -95,9 +95,10 @@ def __init__ (self,
         # Before build with MinGW-W64 generate the python import library
         # with gendef and dlltool according to the MingW-W64 FAQ.
         # Use the MinGW-W64 provided msvc runtime import libraries.
+        # The mingwpy package deploys it's own import libraries.
         # Don't call build_import_library() and build_msvcr_library.
 
-        if 'MinGW-W64' not in str(out_string):
+        if 'MinGW-W64' not in str(out_string) and 'mingwpy' not in str(out_string):
```

Comment:
The strings appear with either vanilla mingw-w64: ```MinGW-W64``` or with mingwpy: ```mingwpy```. In both cases ```build_import_library()``` shouldn't be called, see comment above.